javascript/modules/client/test/integration/client-erc20/decoding.test.ts

```diff
@@ -0,0 +1,106 @@
+// @ts-ignore
+declare const describe, it, expect;
+
+import { ClientErc20, Context, ContextPlugin } from "../../../src";
+
+import {
+  IMintTokenParams,
+  IPluginSettings,
+} from "../../../src/internal/interfaces/plugins";
+import { bytesToHex } from "@aragon/sdk-common";
+import { contextParamsLocalChain } from "../constants";
+
+describe("Client ERC20 decoders", () => {
```

Comment:
Let's have a common parent for the 4 blocks

```suggestion
describe("Client ERC20", () => {
```